scripts/deploy-gh-pages

```diff
@@ -1,3 +0,0 @@
-#!/bin/bash
-sh ./scripts/bundle
-node_modules/.bin/push-dir --dir=dist --branch=gh-pages
```

Comment:
this command will still be useful once the gulp will be able to create a production build in a folder called `dist`, you can re-use it.